src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,158 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> headElement;
+    private Node<T> tailElement;
+    private int size;
+
+    private static class Node<T> {
+        private T value;
+        private Node<T> prev;
+        private Node<T> next;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
+            this.prev = prev;
+            this.value = value;
+            this.next = next;
+
+        }
+    }
+
```

Comment:
it is desirable to be at the very bottom

Reply:
Done